packages/subgraph/schema.graphql

```diff
@@ -888,16 +888,24 @@ type MultisigPlugin implements IPlugin @entity {
 }
 
 type MultisigApprover @entity {
+  """
+  Current Approver of the multisig
+  """
   id: ID! # plugin_address + member_address
   address: String # address as string to facilitate filtering by address on the UI
-  proposals: [MultisigProposalApprover!]! @derivedFrom(field: "approver")
```

Comment:
if we remove this field, won't be able to get the proposals by approver. 

I guess it is removed because `MultisigApprover` was removed from  `MultisigProposalApprover` but think shouldn’t lose the ability to get all the proposals of an approver. 